roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/download-rhel-images.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "Download rhel 8.6 image"
```

Comment:
Can't the RHEL image downloaded by the cluster be used instead? It should not be necessary to do this manually anymore.

Reply:
@0xFelix There are two issues with that, let me know if you have a solution for (a):

(a) The name changes with every release. Kind of tricky to target a moving target.
(b) Only certain (even) releases are certified for/by SAP